client/src/components/survey/SurveyPage1.jsx

```diff
@@ -23,8 +23,8 @@ const useStyles = makeStyles(() => ({
     justifyContent: 'center',
     marginLeft: 'auto',
     marginRight: 'auto',
-    maxWidth: '992px',
```

Comment:
See my comment.

Reply:
I agree! it looks weird.
I didn't look into it  😅 I was trying to make this up and running so from now on everything follow the same sizing and fonts
If you saw discord we were talking about finish what we have now and start work on making such improvement